solidity/contracts/VerifierModule.sol

```diff
@@ -228,6 +228,18 @@ contract VerifierModule is IVerifierModule {
     address[] memory _newOwners = _proposedSettings.owners;
     bool _hasUpdatedThreshold;
 
+    // If all we need to do is a swap the rest of the logic is not needed and we can just call the swapOwner function
+    if (_oldOwners.length == 1 && _newOwners.length == 1 && _oldOwners[0] != _newOwners[0] && _newThreshold == 1) {
```

Comment:
I am not sure if `_newThreshold == 1` is needed, if there is only 1 owner then threshold is always gonna be 1, right?